instance/models/openedx_appserver.py

```diff
@@ -142,21 +149,42 @@ def set_field_defaults(self):
         # assert that it isn't set because if a ValidationError occurred, this method could be
         # called multiple times before this AppServer is successfully created.
         self.configuration_settings = self.create_configuration_settings()
+        self.lms_user_settings = self.create_lms_user_settings()
         super().set_field_defaults()
 
+    def default_playbook(self):
+        """
+        Return a Playbook instance for the standard configuration playbook.
+        """
+        return Playbook(
+            source_repo=self.configuration_source_repo_url,
+            requirements_path='requirements.txt',
+            playbook_path='playbooks/{}.yml'.format(self.RUN_ROLE_PLAYBOOK),
+            version=self.configuration_version,
+            variables=self.configuration_settings,
+        )
+
+    def lms_user_creation_playbook(self):
+        """
+        Return a Playbook instance for creating LMS users.
+        """
+        return Playbook(
+            source_repo=self.configuration_source_repo_url,
+            requirements_path='requirements.txt',
```

Comment:
Too bad this will (presumably) set up a whole new virtualenv that's identical to the one already used for the first playbook. I guess that separation is important though.